core/templates/services/speech-synthesis-chunker.service.spec.ts

```diff
@@ -157,8 +157,11 @@ describe('Speech Synthesis Chunker Service', () => {
 
     beforeEach(() => {
       spyOn(window, 'SpeechSynthesisUtterance').and.returnValue(
-        // @ts-ignore mock doesn't have all property and methods of a native
-        // SpeechSynthesisUtterance.
+        // There is a typescript error here because the type of
+        // mockSpeechSynthesisUtteran doesn't match the actual
```

Comment:
Spelling -- please proofread.

Also, in what way do they not match -- please explain. (The point of this is to be clear to the reader about why we can't just make them match.)

Reply:
done